masterlist.yaml

```diff
@@ -394,43 +400,61 @@ plugins:
   - name: 'DLCRobot.esm'
     group: *dlcGroup
     dirty:
-      - <<: *dirtyPlugin
-        crc: 0xD69027EA
-        util: 'FO4Edit v3.2.33'
-        itm: 45
+      - <<: *quickClean
+        crc: 0xD69027EA # v1.10.114.0
```

Comment:
I generally prefer to have the version on the line above the construct, possibly with explanatory notes. Ie.:
```yaml
      # v1.10.114.0, before cleaning
      - <<: *quickClean
        crc: 0xD69027EA
        …
      # v1.10.114.0, after initial cleaning(?)
      - <<: *quickClean
        crc: 0x1954F90B
        …
```

Just a comment though. Not something I'll hold up the PR over. :)

Reply:
I updated them anyway.